includes/_top.htm

```diff
@@ -150,6 +150,48 @@
 							</li>
 						</ul>
 					</li>
+					<li class="top-nav-item" role="menuitem">
+						<a href="#" class="top-nav-item-anchor" aria-haspopup="true" aria-expanded="false" aria-controls="nav-community-menu">
+							Apache Software
+						</a>
+						<ul class="top-nav-menu" aria-hidden="true" role="menu" id="nav-community-menu" title="Apache Homepage">
+							<li class="top-nav-menu-item" role="menuitem">
+								<a class="top-nav-anchor" tabindex="-1" href="https://www.apache.org/" target="_blank">
+									Apache Homepage
+								</a>
+							</li>
+							<li class="top-nav-menu-item" role="menuitem">
+								<a class="top-nav-anchor" tabindex="-1" href="https://www.apache.org/licenses/">
+									License
+								</a>
+							</li>
+							<li class="top-nav-menu-item" role="menuitem">
+								<a class="top-nav-anchor" tabindex="-1" href="https://www.apache.org/events/current-event">
+									Events
+								</a>
+							</li>
+							<li class="top-nav-menu-item" role="menuitem">
+								<a class="top-nav-anchor" tabindex="-1" href="https://www.apache.org/foundation/sponsorship.html" target="_blank">
+									Sponsorship
+								</a>
+							</li>
+							<li class="top-nav-menu-item" role="menuitem">
+								<a class="top-nav-anchor" tabindex="-1" href="https://www.apache.org/foundation/thanks.html">
+									Sponsors
+								</a>
+							</li>
+							<li class="top-nav-menu-item" role="menuitem">
+								<a class="top-nav-anchor" tabindex="-1" href="https://www.apache.org/security/">
```

Comment:
The Apache policy allows to link to a project specific page, and we already have this at https://kafka.apache.org/project-security. I think using that would be better that linking to the overarching Apache security page.